rfcs/0009_Unified_Static_Memory_Planning.md

```diff
@@ -0,0 +1,473 @@
+    Feature Name: Unified Static Memory Planner
+    Start Date: 2021 June 1
+    RFC PR: #0009
+    GitHub Issue: https://github.com/apache/tvm/issues/8404
+
+# Background
+
+Currently, given a ML model primarily TVM will generate two main artifacts :
+
+* A1 : Description of the sequential execution of operators :
+  1. If the "executor" is "graph", this would be a JSON
```

Comment:
Sorry, Im not sure that I follow. Do you mean 'define' or 'specify'?

Reply:
sorry i mean that by `"executor"` here, you mean the value of the `"executor"` key in MLF `metadata.json`. However, that's a micro-specific (for now) artifact. To make the RFC easier to follow, suggest either including a pointer so that a reader could determine this, or just saying something less specific like "If building for the graph executor."